test.gd

```diff
@@ -3,33 +3,189 @@ extends SceneTree
 # To run this script
 # godot -s test.gd
 
-const NUMBER_OF_TEST = 500000
+const NUMBER_OF_TESTS = 500000
+const NUMBER_OF_OBJECTS = 50000  # enough to test and to not run out of memory
 
-const uuid_util = preload('res://uuid.gd')
+var uuid_util = preload('uuid.gd')
```

Comment:
Wondering why switch between `const` to `var`?

Reply:
It seems that some behavior changes in 4.0 made using `(pre)load()` with `const` impossible. At least it wasn't working for me and `godot -s` would not give me a concise error message.